public/app/features/alerting/unified/components/contact-points/ContactPoints.v1.test.tsx

```diff
@@ -400,7 +400,8 @@ describe('Receivers', () => {
     // see that we're back to main page and proper api calls have been made
     await ui.receiversTable.find();
     expect(mocks.api.updateConfig).toHaveBeenCalledTimes(1);
-    expect(mocks.api.fetchConfig).toHaveBeenCalledTimes(3);
+    expect(mocks.api.fetchConfig).toHaveBeenCalledTimes(1);
```

Comment:
This bothers me a little bit because it should have been called three times (once for list view, once for detail view and once after invalidation).

Manual testing seems to work fine though ¯\_(ツ)_/¯